spyder/app/mainwindow.py

```diff
@@ -2375,40 +2376,75 @@ def report_issue(self, traceback=""):
         revision = ''
         if versions['revision']:
             revision = versions['revision']
+        reminder_message = (
+            "**PLEASE READ:** Before reporting here, *please* carefully "
+            "read our **Troubleshooting Guide**: <{0!s}> and search the issue "
+            "page for your error/problem, as most "
+            "posted bugs are duplicates or easy fixes.\n\n"
+            "Also, if you do post here, please describe step by step in "
+            "detail what you were doing when the error occurred. "
+            "Otherwise, your issue will likely be closed after 7 days. "
+            "Thanks.\n\n"
+            "**Note:** Delete this message before submitting."
+            ).format(__trouble_url__)
+        if traceback:
+            reminder_message = ("IMPORTANT: Read troubleshooting guide at "
+                                "{0!s} before posting!"
+                                ).format(__trouble_url_short__)
+        else:
+            traceback = "PASTE HERE"
         issue_template = """\
-## Description
+%s
 
-**What steps will reproduce the problem?**
+## What steps reproduce the problem?
 
 1. 
 2. 
 3. 
 
-**What is the expected output? What do you see instead?**
-
-
-**Please provide any additional information below**
-
+## Paste Traceback/Error Below
+(from `View > Panels > Internal Console`)

```

Comment:
I don't like this. The traceback should be copied automatically to the clipboard when users press *Send to Github* and they'd only need to press `Ctrl+V` or do a browser `Paste` here to paste it.

Besides, when the error dialog is active, tracebacks are not pasted to the internal console.

Reply:
Okay, I will implement that as requested. My concern is that users, following previous behavior trends, will as often as not fail to copy the traceback and thus we'll get nothing, but I suppose there is only one way to find out...

Reply:
Or will #5346 implement it instead? If so, should to be done here (other than removing the ``(from View > ...) line)?